src/controllers/LikeController.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable class-methods-use-this */
+import Response from '../utils/response';
+import LikeService from '../repositories/LikeRepository';
+
+class LikeController {
+  /**
+   * like or unlike accomodation facility
+   * @param {object} req accommodation
+   * @param {object} res response
+   * @param {object} next next return error
+   * @returns {object} response object
+   */
+  static async LikeOrUnlike(req, res, next) {
+    let accommodation;
+    let alreadyLiked;
+    let Likes;
+    try {
+      const { userData } = req;
+      const AccommodationId = Number(req.params.id);
+
+      accommodation = await LikeService.checkAccommodation(AccommodationId);
+      if (!accommodation) {
+        const response = new Response(res, 404, 'accommodation not found');
+        return response.sendErrorMessage();
+      }
+      Likes = await LikeService.accommodationLikes(AccommodationId);
+      alreadyLiked = await LikeService.CountLikes({
+        user_id: userData.id,
+        accommodation_id: AccommodationId
+      });
+      if (alreadyLiked) {
+        await LikeService.unLike({ user_id: userData.id, accommodation_id: AccommodationId });
+        return res.status(201).json({ status: 201, message: `YOu have unliked ${accommodation.name}`, likes: Likes - 1 });
```

Comment:
I believe you guys have a utility function e.g `const response = new Response(res, 404, 'accommodation not found');` that does this, why are we still doing this?